activiti-engine/src/main/java/org/activiti/engine/impl/ProcessDefinitionQueryImpl.java

```diff
@@ -149,6 +150,14 @@ public ProcessDefinitionQueryImpl processDefinitionKey(String key) {
         return this;
     }
 
+    public ProcessDefinitionQueryImpl processDefinitionKeys(Set<String> keys) {
```

Comment:
IMO the return type should be the interface instead of the implementation.

Reply:
Ignore my previous comment on this. It's ok to have the implementation here (`ProcessDefinitionQueryImpl`), the important is to have the interface in `ProcessDefinitionQuery`. For testing purposes it might be interesting to have the implementation type here.